cypress/integration/integration_spec.js

```diff
@@ -142,37 +205,21 @@ describe('user', () => {
     });
 });
 
-const loremLong = `Lorem, ipsum dolor sit amet consectetur adipisicing elit. Praesentium quidem nihil, similique voluptatibus tempore quasi, cumque laborum officia voluptatem laboriosam tempora.
-
-- Repudiandae doloremque necessitatibus
-- Laudantium repellendus
-- Soluta neque consequatur tenetur maiores.
-
-Enim provident necessitatibus ipsa ad autem aliquam ducimus minima delectus exercitationem, minus blanditiis molestias quas eaque ullam ab aperiam assumenda.`;
-
-describe.skip('awards for all', function() {
-    it('happy path application', () => {
-        const eligibilityStep = () => {
-            cy.getByLabelText('Yes').click();
-            cy.getByText('Continue').click();
-        };
+describe('awards for all', function() {
```

Comment:
This is the main change to these tests. Including the awards for all test rather than skipping it. It only goes part way through the form but even with this addition (and a little streamlining of some of the other tests) the CI test run is still <5 minutes, with the integration tests taking about 1m30 of that so I think we can get some decent coverage of the form here without bloating the build time too much.